x-pack/plugins/security_solution/public/management/pages/policy/view/policy_advanced.tsx

```diff
@@ -137,7 +158,18 @@ const PolicyAdvanced = React.memo(
       <>
         <EuiFormRow
           fullWidth
-          label={configPath.join('.')}
+          label={
+            <EuiFlexGroup>
+              <EuiFlexItem>{configPath.join('.')}</EuiFlexItem>
+              {documentation === '' ? (
```

Comment:
Just FYI - a shorter way to do this would be:

```jsx
{
    documentation &&
    (<EuiFlexItem>
        <EuiIconTip content={documentation} position="right" />
     </EuiFlexItem>)
}
```

Reply:
i'm surprised that `documentation` evaluates to `false` if it's an empty string.

thanks for the tip. fixed: dd6ce7e039ebcdf2ae7370c2b897c73ac03f8571